app/client/components/identity/idapi/user.ts

```diff
@@ -49,6 +49,8 @@ interface UserAPIResponse {
   };
 }
 
+type UserFromAPIResponse = UserAPIResponse["user"];
```

Comment:
What do you think about extracting `UserAPIResponse.user` into its own type and using it inside `UserAPIResponse` instead, as a way to improve clarity of this type hierarchy?